src/library/objects/Quest.js

```diff
@@ -96,20 +96,28 @@ Quest Type:
 		}
 		return "";
 	};
-	
+
 	/* INCREMENT
 	Add one to tracking progress
 	@param {number} reqNum - index of counter type, mainly for Bw1. Default: 0
 	@param {number} amount - progress amount to be increased. Default: 1
 	@param isAdjustingCounter - if true, prevent recursively inc on specific quest itself
 	------------------------------------------*/
-	KC3Quest.prototype.increment = function(reqNum, amount, isAdjustingCounter){
+	KC3Quest.prototype.increment = function(reqNum=0, amount=1, isAdjustingCounter=false){
 		var self = this;
 		var isIncreased = false;
+		if (! ConfigManager.spCounterAdjust) {
+			if (this.tracking && this.isSelected()) {
+				if (this.tracking[reqNum][0] + amount <= this.tracking[reqNum][1]) {

```

Comment:
@sinsinpub I didn't see that line coming from someone that agreed to use es6 for this project?

by the way how buggy is es6? is that even worse than a buggy counter that needs to be fixed at least 3 times?

do you really know that a program first get parsed before ever executing? show me how to execute an ill-syntaxed source code, I'll be fascinated.

well now at least **everyone** has learned how a pro maintains open source projects and understands about programs. I'm sure to take a note for myself. thanks!